platform/admin/admin-app/src/main/resources/features.xml

```diff
@@ -76,6 +76,7 @@
         <feature>admin-modules-docs</feature>
         <bundle>mvn:ddf.admin.modules/admin-metrics-ui/${project.version}</bundle>
         <bundle>mvn:ddf.admin.modules/admin-security-certificate-ui/${project.version}</bundle>
+        <configfile finalname="${karaf.etc}/org.codice.ddf.security.policy.context.impl.PolicyManager.config" override="false">mvn:ddf.security.policy/security-policy-context/${project.version}/config/default-policy</configfile>
```

Comment:
Does this need to be `"${karaf.etc}/` here? Other `<configfile finalname="` places we use `"/etc/`.

Reply:
I'm actually fairly certain all the others should really be using `${karaf.etc}` as the root, so that the storage location could be overridden. This is how all the karaf provided items work at any rate. At the very least I think that the other listings probably shouldn't have the leading `/` in order to to indicate they are placed in `etc` relative to the karaf process working directory. If we want to maintain consistency I'll change it really quick, otherwise I'll probably leave this

Reply:
I'm fine with leaving this for now. I'll write another ticket to make them all consistent.

Reply:
[DDF-3568](https://codice.atlassian.net/browse/DDF-3568)